source/fundamentals/enterprise-auth.txt

```diff
@@ -278,4 +287,131 @@ mechanism:
 
       .. literalinclude:: /examples/generated/EnterpriseAuthTest.snippet.ldap-mongo-credential.kt
          :language: kotlin
-      
\ No newline at end of file
+
+.. _kotlin-oidc:
+
+MONGODB-OIDC
+~~~~~~~~~~~~
+
+.. important::
+
+   The MONGODB-OIDC authentication mechanism requires {+mdb-server+} v7.0 or later running
+   on a Linux platform.
+
+The following sections describe how to use the ``MONGODB-OIDC``
```

Comment:
S: I don't think this needs to be monospaced, based on the other mentions of MONGODB-OIDC, Kerberos, LDAP
```suggestion
The following sections describe how to use the MONGODB-OIDC
```